packages/console/src/Console.ts

```diff
@@ -16,7 +16,7 @@
  * This class supports most of the methods / accessors log4js uses.
  */
 import { IConsole } from "./doc/IConsole";
-import { TextUtils } from "../../utilities";
+import { TextUtils } from "../../utilities/src/TextUtils";
```

Comment:
Did we remove `TextUtils` exports from `utilities/index.ts`? Looks like we're still doing an `export * from "TextUtils"`